pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -50,24 +50,29 @@ var progSingleton *EbpfProgram
 // LibraryCallback defines the type of the callback function that will be called when a shared library event is detected
 type LibraryCallback func(LibPath)
 
+type libsetData struct {
+	libset      Libset
```

Comment:
seems redundant, as it appears in the mapping

Reply:
Fixed